### PR TITLE
feat: plot_scatterplot and plot_lineplot

### LINKED ISSUE #304: Visualize relationships between variables
### Is your feature request related to a problem? Please describe

There is currently no way to visualize relationships between variables, which is important for data exploration and choosing features for ML. 

### Desired solution

Implement a way to visualize relationships, such as scatterplot and lineplot. 

### Possible alternatives (optional)

A Table function such as seaborn's pairplot could be useful, which takes each combination of two variables in the table and produces (by default) a scatterplot for each pair. This would allow for a quick overview of relationships between variables in a table.

### PR DESCRIPTION
**Pull request by SmiteDeluxe**
Closes #304 .

### Summary of Changes

Implemented bot hplot-scatterplot and plot_lineplot and it's tests. Also fixed a copy/paste error in test for plot_histogram.

### CODE REVIEW

**Review by Marvjowa — DISMISSED** [review]

File: Runtime/safe-ds/safe_ds/plotting/_plot_lineplot.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+def plot_lineplot(table: Table, x: str, y: str) -> None:
+    """
+    Plot two columns against each other in a lineplot.
+
+    Parameters
+    ----------
+    table: Table
+        The table containing the data to plot.
+    x: str
+        The columnname of the column that is to be plotted on the x-Axis.
```
Comment (dorianheidorn) [inline Runtime/safe-ds/safe_ds/plotting/_plot_lineplot.py:16]: column name

Comment (dorianheidorn) [inline Runtime/safe-ds/safe_ds/plotting/_plot_lineplot.py:16]:
```suggestion
        The column name of the column that is to be plotted on the x-Axis.
```

File: Runtime/safe-ds/safe_ds/plotting/_plot_scatterplot.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+def plot_scatterplot(table: Table, x: str, y: str) -> None:
+    """
+    Plot two columns against each other in a scatterplot.
+
+    Parameters
+    ----------
+    table: Table
+        The table containing the data to plot.
+    x: str
+        The columnname of the column that is to be plotted on the x-Axis.
```
Comment (WinPlay02) [inline Runtime/safe-ds/safe_ds/plotting/_plot_scatterplot.py:16]:
```suggestion
        The column name of the column that is to be plotted on the x-Axis.
```

File: Runtime/safe-ds/safe_ds/plotting/_plot_scatterplot.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    y str
+        The columnname of the column that is to be plotted on the y-Axis.
```
Comment (WinPlay02) [inline Runtime/safe-ds/safe_ds/plotting/_plot_scatterplot.py:18]:
```suggestion
        The column name of the column that is to be plotted on the y-Axis.
```

File: Runtime/safe-ds/safe_ds/plotting/_plot_lineplot.py (hunk 1 above)
Comment (dorianheidorn) [inline Runtime/safe-ds/safe_ds/plotting/_plot_lineplot.py:16]:
```suggestion
        The column name of the column that is to be plotted on the x-Axis.
```

File: Runtime/safe-ds/safe_ds/plotting/_plot_scatterplot.py (hunk 2 above)
Comment (dorianheidorn) [inline Runtime/safe-ds/safe_ds/plotting/_plot_scatterplot.py]:
```suggestion
        The column name of the column that is to be plotted on the x-Axis.
```

Comment (dorianheidorn) [inline Runtime/safe-ds/safe_ds/plotting/_plot_scatterplot.py]:
```suggestion
        The column name of the column that is to be plotted on the x-Axis.
```

**Review by dorianheidorn — APPROVED** [review]

File: Runtime/safe-ds/safe_ds/plotting/_plot_lineplot.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+def plot_lineplot(table: Table, x: str, y: str) -> None:
+    """
+    Plot two columns against each other in a lineplot.
+
+    Parameters
+    ----------
+    table: Table
+        The table containing the data to plot.
+    x: str
+        The column name of the column that is to be plotted on the x-Axis.
```
Comment (WinPlay02) [inline Runtime/safe-ds/safe_ds/plotting/_plot_lineplot.py:16]:
```suggestion
        The column name of the column to be plotted on the x-Axis.
```

File: Runtime/safe-ds/safe_ds/plotting/_plot_lineplot.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    y str
+        The column name of the column that is to be plotted on the y-Axis.
```
Comment (WinPlay02) [inline Runtime/safe-ds/safe_ds/plotting/_plot_lineplot.py:18]:
```suggestion
        The column name of the column to be plotted on the y-Axis.
```

File: Runtime/safe-ds/safe_ds/plotting/_plot_scatterplot.py [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+def plot_scatterplot(table: Table, x: str, y: str) -> None:
+    """
+    Plot two columns against each other in a scatterplot.
+
+    Parameters
+    ----------
+    table: Table
+        The table containing the data to plot.
+    x: str
+        The column name of the column that is to be plotted on the x-Axis.
```
Comment (WinPlay02) [inline Runtime/safe-ds/safe_ds/plotting/_plot_scatterplot.py:16]:
```suggestion
        The column name of the column to be plotted on the x-Axis.
```

File: Runtime/safe-ds/safe_ds/plotting/_plot_scatterplot.py [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    y str
+        The column name of the column that is to be plotted on the y-Axis.
```
Comment (WinPlay02) [inline Runtime/safe-ds/safe_ds/plotting/_plot_scatterplot.py:18]:
```suggestion
        The column name of the column to be plotted on the y-Axis.
```

**Review by WinPlay02 — APPROVED** [review]

**Pull request merged**